packages/design-preview/src/components/sidebar.tsx

```diff
@@ -115,6 +125,18 @@ const Sidebar: React.FC< SidebarProps > = ( {
 						</div>
 					</div>
 				) }
+				{ variations &&
```

Comment:
The `variations` is `undefined` when we're still loading the theme details. So, we have to check it first to avoid displaying font pairings first and then changing to the style variations